metrics.go

```diff
@@ -3,47 +3,119 @@
 package goa
 
 import (
+	"regexp"
+	"strings"
+	"sync"
+	"sync/atomic"
 	"time"
 
 	"github.com/armon/go-metrics"
 )
 
-// metriks is the local instance of metrics.Metrics
-var metriks *metrics.Metrics
+const (
+	allMatcher      string = "*/*"
+	allReplacement  string = "all"
+	normalizedToken string = "_"
+)
+
+var (
+	// metriks atomic value storage
+	metriks atomic.Value
+
+	// mu mutex for metriks
```

Comment:
Please remove the mutex, it is not needed anymore (see the [first example](https://golang.org/pkg/sync/atomic/#Value) of sync.Value).